src/analysis/buildTypeWeights.ts

```diff
@@ -372,28 +409,36 @@ function parseTypes(schema: GraphQLSchema, typeWeights: TypeWeightSet): TypeWeig
  *  - validate that the typeWeightsConfig parameter has no negative values (throw an error if it does)
  *
  * @param schema
+ * @param enforceBoundedLists Defaults to false
  * @param typeWeightsConfig Defaults to {mutation: 10, object: 1, field: 0, connection: 2}
  */
 function buildTypeWeightsFromSchema(
     schema: GraphQLSchema,
-    typeWeightsConfig: TypeWeightConfig = defaultTypeWeightsConfig
+    typeWeightsConfig: TypeWeightConfig = defaultTypeWeightsConfig,
+    enforceBoundedLists = false
 ): TypeWeightObject {
-    if (!schema) throw new Error('Missing Argument: schema is required');
+    try {
+        if (!schema) throw new Error('Missing Argument: schema is required');
```

Comment:
This error will get caught be the try catch block and the primary error will be `Error in expressGraphQLRateLimiter when parsing schema object: ${err}`. Is that intended?

Reply:
I think it was yes